crates/sui-bridge/src/server/mod.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use axum::{http::StatusCode, routing::get, Router};
+use std::net::SocketAddr;
+use std::sync::Arc;
+
+use crate::{
+    error::BridgeError,
+    server::handler::{BridgeRequestHandler, BridgeRequestHandlerTrait},
+    types::SignedBridgeAction,
+};
+use axum::{
+    extract::{Path, State},
+    Json,
+};
+
+pub mod handler;
+
+#[cfg(test)]
+pub(crate) mod mock_handler;
+
+pub const APPLICATION_JSON: &str = "application/json";
+
+// Important: the paths need to match the ones in bridge_client.rs
+pub const ETH_TO_SUI_TX_PATH: &str = "/sign/bridge_tx/eth/sui/:tx_hash/:event_index";
+pub const SUI_TO_ETH_TX_PATH: &str = "/sign/bridge_tx/sui/eth/:tx_digest/:event_index";
```

Comment:
for reviewer: this file was moved from `server.rs`, these 3 lines. No need to look at other parts other than these 3 lines